product_pricelist_item_uom/tests/test_product_pricelist_item.py

```diff
@@ -0,0 +1,93 @@
+#  Copyright 2023 Simone Rubino - Aion Tech
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from math import log10
+
+from odoo.tests import Form, TransactionCase
+
+
+class TestProductPricelistItem(TransactionCase):
```

Comment:
I suggest you to inherit from `BaseCommon` class in order to improve tests performances.

https://github.com/odoo/odoo/blob/a877336b4d007c06c9759e4c8a302038bae20a26/odoo/addons/base/tests/common.py#L19

Reply:
Thanks for having a look!
Didn't know that one, done